### DISCUSSION TITLE
Max claim size

### DISCUSSION THREAD

**Discussion by elaineandsparky** [GriefPrevention/GriefPrevention#1112] [Ideas]
As far as I can see there is no max limit to one claim size?

There is a minimum claim size and a max number of claims but no max claim size?
Or a max total number of claim blocks per person would be ok. As there is only a max occured amount not max purchased amount.

I have players wanting to try and claim the whole world or very large chunks of it just to stop others from building which would ruin the game for others yet I can't see a way of blocking this?

**Comment by RoboMWM** [comment]
Purchase amount is designed to be infinite. I have an addon that has it mapped to accrued blocks but idk if it works. BuyAccruedBlocks if you wanna look for it, should be on spigot or bukkit or something.
